README.rst

```diff
@@ -60,9 +60,11 @@ Properties are unchanged, so ``conn.prop`` is correct as well as
         conn = await aioodbc.connect(dsn=dsn, loop=loop)
 
         cur = await conn.cursor()
-        await cur.execute("SELECT 42;")
-        r = await cur.fetchall()
-        print(r)
+        await cur.execute("SELECT 42 AS age;")
```

Comment:
42 is not age but [answer to the ultimate query](https://en.wikipedia.org/wiki/Phrases_from_The_Hitchhiker%27s_Guide_to_the_Galaxy#Answer_to_the_Ultimate_Question_of_Life,_the_Universe,_and_Everything_(42))
:)